mne/io/kit/tests/test_kit.py

```diff
@@ -151,10 +156,33 @@ def test_ch_loc():
 
     # test when more than one marker file provided
     mrks = [mrk_path, mrk2_path, mrk3_path]
-    read_raw_kit(sqd_path, mrks, elp_path, hsp_path, preload=False)
+    read_raw_kit(sqd_path, mrks, elp_txt_path, hsp_txt_path, preload=False)
     # this dataset does not have the equivalent set of points :(
     raw_bin.info['dig'] = raw_bin.info['dig'][:8]
     raw_py.info['dig'] = raw_py.info['dig'][:8]
     assert_dig_allclose(raw_py.info, raw_bin.info)
 
+
+def test_hsp_elp():
+    """Test KIT usage of *.elp and *.hsp files against *.txt files
+    """
```

Comment:
FYI our converged convention is now to put these on the same line like `"""Test ... """`